runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr

```diff
@@ -0,0 +1 @@
+../../../../../../antlr
```

Comment:
@pboyer - It is a soft link to the directory containing the Go runtime.
I had a test Go file that I used to test the generate code.
The runtime source needs to be under a `src` directory for `go build`
Git will recreate the link of Unix based system, not sure what happens on Windows.